src/App.js

```diff
@@ -2,13 +2,58 @@ import React from "react";
 import "./App.css";
 
 class App extends React.Component {
-  render() {
-    return (
-      <main>
-        <p>React State Lab</p>
-      </main>
+constructor() {
+  super();
+  this.state = {
+    scoreNow: 0,
+    adding: 1,
+  };
+}
+
+handleClick = () => {
+  this.setState({
+    scoreNow: this.state.scoreNow + this.state.adding,
+  });
+};
+
+forUser = () => {
+  alert("You can't afford that!");
+};
+
+newScore = () => {
+  this.state.adding = this.state.adding + 1;
+  this.setState({ scoreNow: this.state.scoreNow - 10 });
+};
```

Comment:
```suggestion
newScore = () => {
  this.setState({ 
  scoreNow: this.state.scoreNow - 10,
  adding: this.state.adding + 1
   });
};
```
Same problem as above. You're modifying state directly on line 24. Use `this.setState` if you need to update state in React, **do not modify state directly**!!